src/shogun/machine/LinearMachine.cpp

```diff
@@ -22,11 +22,11 @@ CLinearMachine::CLinearMachine(): CMachine()
 	init();
 }
 
-CLinearMachine::CLinearMachine(bool compute_bias): CMachine()
+CLinearMachine::CLinearMachine(bool comput_bias): CMachine()
```

Comment:
dont really like the new name, since it has a typo, but that might be just me being OCD ;)